build_tools/rocm/run_xla.sh

```diff
@@ -37,20 +37,9 @@ echo ""
 echo "Bazel will use ${N_BUILD_JOBS} concurrent build job(s) and ${N_TEST_JOBS} concurrent test job(s) for gpu ${AMD_GPU_GFX_ID}."
 echo ""
 
-# First positional argument (if any) specifies the ROCM_INSTALL_DIR
-if [[ -n $1 ]]; then
-    ROCM_INSTALL_DIR=$1
-else
-    if [[ -z "${ROCM_PATH}" ]]; then
-        ROCM_INSTALL_DIR=/opt/rocm/
-    else
-        ROCM_INSTALL_DIR=$ROCM_PATH
-    fi
-fi
-
 export PYTHON_BIN_PATH=`which python3`
 export TF_NEED_ROCM=1
-export ROCM_PATH=$ROCM_INSTALL_DIR
+export ROCM_PATH="/opt/rocm"
```

Comment:
I was wondering does `/opt/rocm` always point to the correct rocm path, e.g., `/opt/rocm-6.4.3`? Since when? I usually manually specify the rocm path in the local env.

Reply:
Yes, it's will ping to the related rocm version path. I guess since rocm5.7 ?